### ISSUE TITLE
When I add more scenes to my project I cannot build the server as its asks for old scene names.

### ISSUE THREAD

**Issue by jacobjgrant** [suriyun-production/mmorpg-kit-docs#2557]
**Describe the bug**
When I change the scenes in my project and then build server. I start the server and its asking me for the old scene names.

**To Reproduce**
Change scene name from Map_001 to Forest 1

**What should Happen**
If I add more scenes or change the scene name, then when I build the project it should recognise the new scene names not use the old ones.

Are the old scene names stored somewhere in a file that I need to replace once I change the scene name?

**Comment by jacobjgrant** [comment]
![image](https://github.com/user-attachments/assets/f35a424d-4a3a-4e8d-9af6-514f9ce73e06)
![image](https://github.com/user-attachments/assets/1c9e111e-9e41-43ad-9820-6a3251b599ea)

**Comment by jacobjgrant** [comment]
![image](https://github.com/user-attachments/assets/d881351e-689b-4494-89ec-9b900308f6f0)
![image](https://github.com/user-attachments/assets/1fd52e02-d656-40c1-8dec-b44c8430a959)

**Comment by jacobjgrant** [comment]
Ass you can see I have added all the correct scenes in the database and the DatabaseNetworkManager. There must a reference to Map_001 somewhere else

**Comment by moepi2k** [comment]
check the correct name in  ur error. mymap_001 is not added to de building settings

**Comment by jacobjgrant** [comment]
Hi sorry was at work. Ill check now

**Comment by jacobjgrant** [comment]
![image](https://github.com/user-attachments/assets/c50f2afa-41fa-460c-8c24-330e1cd6162f)

**Comment by jacobjgrant** [comment]
Ok so its in my err log file how do I fix this?

**Comment by jacobjgrant** [comment]
I think I fixed it. The Nethraldor map was missing in the map info for Nethraldor. Nethraldor was my starting map (Map_001) . So i am assuming becuase it was not added to the map info, it tried to load up a default scene,. Im assuming that's Map_001

**Comment by insthync** [comment]
Yep, it is having problem after change the scene name later, scene name which store to map info won't updated follow the changed scene name

How I gonna fix it?, I might store scene object in editor then when build or press play, I will make it validate scene name, if it is not correct, then it must change the scene name.

**Comment by khongtuong** [comment]
When you change the scene name, it's necessary to delete and then re-add the renamed scene to the MapInfo ScriptableObject, as well as re-add it to the database script. Additionally, remove the renamed scene from the build and add it again. Sometimes, you may need to delete entries in MapInfo, the build, and the database, close the Unity editor, then reopen it and add everything once more. I've encountered this issue before, and the steps mentioned are my solution, which I hope will be helpful.

**Comment by insthync** [comment]
@khongtuong Just replace a scene into the scene field should work, as I told this have to be fixed

**Comment by Eniotnacram** [comment]
Yes, hard reset is needed for some reason, remove scene from map info, save, add scene again and save. Should work, make sur your server config also have the right name as per your scriptable map info.
